algorithms/C/tree/MaxElementOfTree.c

```diff
@@ -0,0 +1,119 @@
+// Wap to find the maximum(using recursion) element in a binary search tree.
+
+#include<stdio.h>
+#include<stdlib.h>
+
+struct Node{
+    int data;
+    struct Node *left;
+    struct Node *right;
+};
+
+// creating a Node
+struct Node * createNode(int data){
+	struct Node *n;
+    n = (struct Node *)malloc(sizeof(struct Node)); // allocating the memory in the heap
+    n -> data = data; // setting the data
+    n -> left = NULL; // setting left node to be NULL
+    n -> right = NULL; // setting right node to be NULL
+    return n; // returning the created node
+}
+
+//function to insert a node in tree
+struct Node *insert(struct Node *root, int data){
+	if(root==NULL){ // empty tree condition
+		root = createNode(data);
+		return root;
+	}
+	
+	// if tree is not empty (data to be inserted at left subtree)

```

Comment:
It would be better if you use multi line comment here
> // if tree is not empty (data to be inserted at left subtree)
	// data is to be inserted and root->data is value of the root node to be compared with passed data